src/cart/SConscript

```diff
@@ -22,6 +22,25 @@ SRC = ['crt_bulk.c', 'crt_context.c', 'crt_corpc.c',
        'crt_swim.c', 'crt_tree.c', 'crt_tree_flat.c', 'crt_tree_kary.c',
        'crt_tree_knomial.c', 'crt_hlc.c', 'crt_hlct.c']
 
+LIB_SRC = ['crt_utils.c']
+
+##############################
+# Create cart utility library
+##############################
+def build_utility_library():
+    """Build cart utility library"""
+
+    Import('env', 'prereqs')
```

Comment:
Just pass them into the function as they are imported in scons() already.

Reply:
removed